app/assets/stylesheets/general_style.css.sass

```diff
@@ -72,6 +72,10 @@ i.last-updated
   input.prompt-message
     margin: 5px 0
 
+input.invalid
+  border-color: red
+  background-color: #ffeeee
+
```

Comment:
Might want to reuse the [built-in Bootstrap CSS classes for this](http://getbootstrap.com/css/#forms-control-validation)?